### ISSUE TITLE
Thank you. Its working Blender 4.1 also.

### ISSUE THREAD

**Issue by JuhaW** [Claw256/ungrouper#3]
I tried to code also ungroup all nodes and everytime Blender crashed :) I thought it was simple, just select group node and bpy.ops.group_ungroup and do it until there are no group nodes, but no :)

**Comment by sky90038528** [comment]
yeah , blender crashed everytime on 4.2/4.1, support new version plz

**Comment by Claw256** [comment]
@JuhaW @sky90038528 Hiya, thanks for the feedback! I'm currently on vacation, I will be back home in a week, I will fix the add-on when I return home :)
